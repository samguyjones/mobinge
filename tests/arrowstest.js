import React from 'react';
import Arrows from '../src/components/Arrows';
import renderer from 'react-test-renderer';

test('Test Render', () => {
  const style = {
    border: '1px solid black'
  };
  const component = renderer.create(
    <Arrows width="320px"></Arrows>
  );
  let render = component.toJSON();
  expect(render).toMatchSnapshot();
});
