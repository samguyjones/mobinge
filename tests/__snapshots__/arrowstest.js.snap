// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Render 1`] = `
<div
  style={
    Object {
      "position": "relative",
      "width": "320px",
    }
  }
>
  <span
    style={
      Object {
        "left": "0px",
      }
    }
  >
    <a
      className="first-arrow"
      onClick={undefined}
    >
      <img
        src="images/first-arrow.png"
      />
    </a>
    <a
      className="back-arrow"
      onClick={undefined}
    >
      <img
        src="images/back-arrow.png"
      />
    </a>
  </span>
  <span
    style={
      Object {
        "position": "absolute",
        "right": "0px",
      }
    }
  >
    <a
      className="next-arrow"
      onClick={undefined}
    >
      <img
        src="images/next-arrow.png"
      />
    </a>
    <a
      className="last-arrow"
      onClick={undefined}
    >
      <img
        src="images/last-arrow.png"
      />
    </a>
  </span>
</div>
`;
